support-workers/src/test/scala/com/gu/zuora/subscriptionBuilders/DigitalSubscriptionBuilderSpec.scala

```diff
@@ -96,6 +105,7 @@ class DigitalSubscriptionBuilderSpec extends AsyncFlatSpec with Matchers {
         )))
       })),
     SANDBOX,
+    giftCodeGeneratorService,
```

Comment:
funny that we need a gift code generator for a corporate sub, would be better if the types were narrower

Reply:
The crazy world of Digital Subs!